python-sdk/tutorials/nuimages_tutorial.ipynb

```diff
@@ -196,7 +198,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "nuim.render_image(sd_token_camera, with_category=True, with_attributes=False, box_line_width=0, render_scale=2)"
+    "nuim.render_image(sd_token_camera, with_annotations='all',\n",
```

Comment:
add font_size field just to let viewers know that we have that (Update: font_size didn't work for me :( )

Reply:
when i tried to set `none` it was still quite slow in rendering which surprised me.

Reply:
surprising too that the box_line_width=0 still shows a box line... is this expected?

Reply:
sorry i didn't read the description about box_line_width... maybe you can make it -1 for adjusting to rendering size and box_line_width=0 for no box line at all

Reply:
@venice-erin, interestingly, `draw.rectangle(bbox, outline=color, width=0) will still produce a line (albeit a v thin line), but PIL will draw a line even if the width is specified to be zero

Reply:
Regardless, I've changed it to -1 for adjusting to rendering size